### DISCUSSION TITLE
Fonctionnement mode automatique

### DISCUSSION THREAD

**Discussion by Nuyonuyonoina** [AmauriC/tarteaucitron.js#1070] [Q&A] [answered]
Bonjour,
Désolée pour la question en mode newbie, mais je n'arrive pas à trouver de tuto sur la question. J'utilise TAC en mode pro avec le plugin WordPress et le mode automatique activé. Tout fonctionne impec pour les scripts qui sont sur toutes les pages (par ex. GA), mais dès que l'on arrive sur une page avec un autre script, le bandeau surgit de nouveau. Par exemple, l'internaute arrive sur la page d'accueil où le mode automatique ne trouve que Google Analytics. Il accepte ou refuse, peu importe. Puis il va sur une page où se trouve une vidéo YouTube. Nouveau surgissement du bandeau pour accepter ou refuser. Puis il va sur la page de contact et rebelote pour Google Maps. Je pense qu'il doit exister un moyen d'indiquer dès le départ que le site utilise GA / YT / GMaps et d'obtenir le consentement une bonne fois pour toutes, n'est-ce pas ? Parce que ces surgissements intempestifs sont un poil lassants.
Merci d'avance pour m'aiguiller dans la bonne direction.

**Comment by AmauriC** [comment, marked as answer]
Salut,

C'est une question légitime ^^

Il faut ajouter sur le site un tag Javascript pour forcer la demande des services qui ne sont pas présent partout:

```
<script>
(tarteaucitron.job = tarteaucitron.job || []).push('youtube');
(tarteaucitron.job = tarteaucitron.job || []).push('googlemaps');
</script>
```

Les ids sont dispo dans le guide de la configuration manuelle https://tarteaucitron.io/fr/install/ (il ne faut que la ligne `.push` inutile d'ajouter les éventuels paramètres).

**Reply by Nuyonuyonoina** [reply]
Top ! Merci @AmauriC ! Tout s'éclaire :-D

**Comment by Nuyonuyonoina** [comment]
Rien à voir, mais un plugin pour Joomla! est-il dans les tuyaux ? Ce serait génial !

**Reply by AmauriC** [reply]
Non parce que je suis déjà pas mal occupé avec la v pro et la base de tarteaucitron 😬

Par contre si un volontaire s'y mets, je suis dispo pour aider :)

**Reply by Nuyonuyonoina** [reply]
Compris ! Je vais lancer un appel au volontariat dans la communauté.
